app/uploaders/banner_image_uploader.rb

```diff
@@ -0,0 +1,8 @@
+class BannerImageUploader < CarrierWave::Uploader::Base
+  include CarrierWave::Compatibility::Paperclip
+
+  # Define valid extensions for site banner image
+  def extension_white_list
+    %w(jpg jpeg png gif tif tiff)
```

Comment:
TIFF isn't a format that most web browsers can display natively.  Even if it were so, it's not a good choice due to the file size.